grakn-test/test-integration/src/test/java/ai/grakn/test/rule/EngineContext.java

```diff
@@ -269,4 +264,16 @@ private static void setRestAssuredUri(GraknConfig config) {
     public JedisPool getJedisPool() {
         return jedisPool;
     }
+
+    private GraknEngineServer createGraknEngineServer(RedisWrapper redisWrapper) throws IOException {
+        EngineID id = EngineID.me();
+        GraknEngineStatus status = new GraknEngineStatus();
+        MetricRegistry metricRegistry = new MetricRegistry();
+        IndexStorage indexStorage =  RedisIndexStorage.create(redisWrapper.getJedisPool(), metricRegistry);
+        CountStorage countStorage = RedisCountStorage.create(redisWrapper.getJedisPool(), metricRegistry);
+        LockProvider lockProvider = new JedisLockProvider(redisWrapper.getJedisPool());
+        EngineGraknTxFactory engineGraknTxFactory = EngineGraknTxFactory.create(lockProvider, config);
+        return GraknEngineServerFactory.createGraknEngineServer(id, spark, status, metricRegistry, config, redisWrapper,
+                indexStorage, countStorage, lockProvider, Runtime.getRuntime(), Collections.emptyList(), engineGraknTxFactory);
+    }
```

Comment:
This sort of stuff probably suggests we do need a builder... but it's OK for the moment in my opinion!